docs-src/content/getting-started/streaming.md

```diff
@@ -101,12 +107,213 @@ audio = open('test.wav', 'rb')
+    mic.stream = stream
+
+    err = mic.stream.Start()
+    if err != nil {
+        return nil, err
+    }
+
+    return &mic, nil
+}
+
+// Read copies N bytes into the passed buffer from the microphone audio buffer
+// where N is the buffer size passed to `cubic.NewClient`. Also, to be compatible
+// with the cubic client, it returns two things : an int representing the number of
+// bytes copied and an error.
+func (mic *Microphone) Read(buffer []byte) (int, error) {
+    err := mic.stream.Read()
+    if err != nil {
+        return 0, err
+    }
+    byteBuffer := new(bytes.Buffer)
+    err = binary.Write(byteBuffer, binary.LittleEndian, mic.buffer)
+    if err != nil {
+        return 0, err
+    }
+    copy(buffer, byteBuffer.Bytes())
+    return len(buffer), nil
+}
+
+// Close shuts the microphone stream down and cleans up
+func (mic *Microphone) Close() {
+    mic.stream.Stop()
+    mic.stream.Close()
+    portaudio.Terminate()
+}
+
+func main() {
+
+    bufferSize := uint32(8192)  // 8192 bytes
+    client, err := cubic.NewClient(serverAddr, cubic.WithInsecure(), cubic.WithStreamingBufferSize(bufferSize))
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer client.Close()
+
+    modelResp, err := client.ListModels(context.Background())
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    // Use the first available model
+    model := modelResp.Models[0]
+
+    // Create microphone stream
+    mic, err := NewMicrophone(model.Attributes.SampleRate, bufferSize)
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer mic.Close()
+
+    cfg := &cubicpb.RecognitionConfig{
+        ModelId: model.Id,
+    }
+
+    // Define a callback function to handle results
+    resultHandler := func(resp *cubicpb.RecognitionResponse) {
+        for _, r := range resp.Results {
+            if r.IsPartial {
+                fmt.Print("\r", r.Alternatives[0].Transcript) // print on same line
+            } else {
+                fmt.Println("\r", r.Alternatives[0].Transcript) // print and move to new line
+            }
+        }
+    }
+
+    err = client.StreamingRecognize(context.Background(), cfg, mic, resultHandler)
+    if err != nil {
+        log.Fatal(err)
+    }
+}
+```
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+This example requires the [pyaudio](http://people.csail.mit.edu/hubert/pyaudio/) 
+module to stream audio from a microphone. Instructions for installing pyaudio for 
+different systems are available at the link. On most platforms, this is simply `pip install pyaudio`.
+
+``` python
+import cubic
+import pyaudio
+
+serverAddress = '127.0.0.1:2727'
+
+client = cubic.Client(serverAddress)
+
+# get list of available models
+modelResp = client.ListModels()
+for model in modelResp.models:
+    print("ID = {}, Name = {}".format(model.id, model.name))
+
+# use the first available model
+model = modelResp.models[0]
+
+cfg = cubic.RecognitionConfig(
+    model_id = model.id
+)
+
+# client.StreamingRecognize takes any binary
```

Comment:
This comment seems to be wrapped a bit too short. Can you check and wrap it at col 80?